dagshub/__init__.py

```diff
@@ -1,3 +1,4 @@
+__version__ = "0.2.1"
```

Comment:
Do we need to update this line manually on every release?

Reply:
was it like that before?

Reply:
Yep, we need to update it on every release.
Before that it was 2 places where it needed to be updated (version.py + in setup.py), now it will be only one place